envVariables.js

```diff
@@ -35,6 +35,10 @@ let enviromentVariables = {
   "KAFKA_URL" : {
     "message" : "Required",
     "optional" : false
+  },
+  "SUNBIRD_SERVICE_URL" : {
```

Comment:
@VISHNUDAS-tunerlabs already USER_SERVICE_URL is there in code use that. we have added this for profile read API